### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+The *fledge-filter-fft* filter is designed to accept some periodic data such as a sample electrical waveform, audio data or vibration data and perform a Fast Fourier Transform on that data to supply frequency data about that waveform.
+
+Data is added as a new asset which is named as the sampled asset with " FFT" append. This FFT asset contains a set of data points that each represent the a band of frequencies, or as a frequency spectrum in a single array data point. The band information that is returned by the filter can be chosen by the user. The options available to represent each band are; 
+
+  - the average in the band, 
+
+  - the peak, 
+    
+  - the RMS
```
Comment (ashish-jabble) [inline docs/index.rst:16]: ,

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    - **Frequency Bands**: The number of frequency bands to divide the resultant FFT output into
+
+    - **Band Prefix**: The prefix to add to the data point names for each band in the output
+
+    - **No. of Samples per FFT**: The number of input samples to use. This must be a power of 2.
+
+    - **Low Frequency Reject %**: A percentage of low frequencies to discard, effectively reducing the range of frequencies to examine
+
+    - **High Frequency Reject %**: A percentage of low frequencies to discard, effectively reducing the range of frequencies to examine
```
Comment (ashish-jabble) [inline docs/index.rst]: high frequencies

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  - Configure your FFT filter
+
+    - **Asset to analysis**: The name of the asset that will be used as the input to the FFT algorithm.
+
+    +---------+
+    | |fft_2| |
+    +---------+
+
+    - **Result Data**: The data that should be returned for each band. This may be one of average, sum, peak, RMS or spectrum. Selecting average will return the average amplitude within the band, sum returns the sum of all amplitudes within the frequency band, peak the greatest amplitude and RMS the root mean square of the amplitudes within the band. Setting the output type to be spectrum will result in the full FFT spectrum data being written.  Spectrum data however can not be sent to all north destinations as it is not supported natively on all the systems Fledge can send data to.
```
Comment (ashish-jabble) [inline docs/index.rst]: `rms and spectrum` - dropdown options are in lowercase

Comment (MarkRiddoch) [reply docs/index.rst]: Really they should not be lower cased, should probably fix that in the code. However for now we make the docs match.

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**